minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/block/forward/FWBlock.java

```diff
@@ -494,11 +494,19 @@ public void renderItem(ItemRenderType type, ItemStack item, Object... data) {
 	@Override
 	public float getExplosionResistance(Entity expEntity, World world, int x, int y, int z, double explosionX, double p_explosionresistance, double explosionY) {
 		// TODO: Maybe do something withPriority these parameters.
-		return (float) getBlockInstance(world, new Vector3D(x, y, z)).getResistance() * 30;
+		return (float) getBlockInstance(world, new Vector3D(x, y, z)).getResistance() * 6;
```

Comment:
wohoo, magic constants

Reply:
This number was calculated from the blast resistance of Stone, which requires exactly one cubic meter of TNT to get blown up.

1. During construction, the `setResistance` method is called on `minecraft:stone` with a value of `10`.
2. The `setResistance` method multiplies that by `3` and assigns the result to the `blockResistance` instance variable.
3. Finally, the `getExplosionResistance` method divides the `blockResistance` instance variable by `6` and returns the result.

From this we see that `minecraft:stone`’s final blast resistance is 6.